cluster/ubuntu/util.sh

```diff
@@ -306,10 +306,7 @@ function kube-up() {
   source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE:-config-default.sh}"
 
   # downloading tarball release
-  if [[ -d "${KUBE_ROOT}/cluster/ubuntu/binaries" ]]; then
-    rm -rf "${KUBE_ROOT}/cluster/ubuntu/binaries"
-  fi
-  "${KUBE_ROOT}/cluster/ubuntu/download-release.sh"
+  [[ -d "${KUBE_ROOT}/cluster/ubuntu/binaries" ]] || "${KUBE_ROOT}/cluster/ubuntu/download-release.sh"
```

Comment:
How can I drop these code? Do I need create another pull request?

Shall I change the code to skip download if not modified? We, in China, need a over 15 minutes to download the file. If something wrong when running `kube-up.sh`, we have try again and again. So it not acceptable to always download the file.